### PR TITLE
RMF: Add support for PZ-90.11 and GSK-2011 coordinate systems

### PR DESCRIPTION
**Pull request by drons**

### CODE REVIEW

File: gdal/ogr/ogr_srs_panorama.cpp [hunk 1]
```diff
@@ -436,6 +438,18 @@ OGRErr OGRSpatialReference::importFromPanorama( long iProjSys, long iDatum,
             oGCS.importFromEPSG( aoDatums[iDatum] );
             CopyGeogCSFrom( &oGCS );
         }
+        else if( iEllips == PAN_ELLIPSOID_GSK2011 )
+        {
+            OGRSpatialReference oGCS;
+            oGCS.importFromEPSG( 7682 );
```
Comment (rouault) [inline gdal/ogr/ogr_srs_panorama.cpp]: 7682 and 7680 are the codes for the Geographic 3D CRS, not the 2D ones. Is this intended ?

Comment (drons) [reply gdal/ogr/ogr_srs_panorama.cpp]: Oo, yes! Thanks!

File: gdal/ogr/ogr_srs_panorama.cpp [hunk 2]
```diff
@@ -436,6 +438,18 @@ OGRErr OGRSpatialReference::importFromPanorama( long iProjSys, long iDatum,
...
         }
+        else if( iEllips == PAN_ELLIPSOID_GSK2011 )
+        {
+            OGRSpatialReference oGCS;
+            oGCS.importFromEPSG( 7683 );
+            CopyGeogCSFrom( &oGCS );
+        }
+        else if( iEllips == PAN_ELLIPSOID_PZ90 )
+        {
+            SetGeogCS( "PZ-90.11", "Parametry_Zemli_1990_11",
```
Comment (drons) [inline gdal/ogr/ogr_srs_panorama.cpp]: Unfortunately, I can't find valid EPSG code for `PZ-90.11` 2D GeogCS .
Thus, I define GeogCS explicitly.

**Comment by drons** [comment]
Thanks!

**Pull request merged**